### ISSUE TITLE
The aspect-ratio-notation fix spells its solidus with a space on each side, and the configuration has no say in it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#550]
### What minimal example or steps are needed to reproduce the bug?

```css
a { aspect-ratio: 16/9; }
b { aspect-ratio: 2; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": [
		"@stylistic/stylelint-plugin"
	],
	"rules": {
		"@stylistic/aspect-ratio-notation": "ratio"
	}
}
```

### What did you expect to happen?

The ratio the fix writes to be spelled the way the file spells the ratios it already has, or the way the configuration asks — and, once there is a rule about the whitespace around a solidus, the way that rule asks whatever order the configuration lists the two in.

### What actually happened?

One run of `--fix` leaves:

```css
a { aspect-ratio: 16/9; }
b { aspect-ratio: 2 / 1; }
```

The first declaration keeps the spelling the file gave it, since the rule leaves an already written ratio alone. The second is written with a space on each side of the solidus, and there is nothing to configure about it: `lib/rules/aspect-ratio-notation/index.ts:94` is the one place the rule writes a solidus, and the text is a constant.

```ts
if (writesHeight && !height) edits.push({ start: width.sourceEndIndex, end: width.sourceEndIndex, text: ` / ${expectedHeight}` })
```

So a stylesheet whose author writes `16/9` is handed `2 / 1` by the fix, and no configuration of this plugin brings it back: of the 208 configurations the plugin offers, none writes a solidus tight or spaced either way, which is #548.

### Do you have a proposal to fix the bug?

Take the form from the neighbours rather than from a constant: `whitespaceAsked` once per run, with ` / ` as the fallback, so that the solidus is spelled the way `value-slash-space-before` and `value-slash-space-after` ask wherever the configuration lists them, and exactly as it is spelled today wherever it lists neither. That wants #548, since until those rules exist there is nothing to read, and #549, which is the loop itself.

The fix is one line, and the message needs no separate change: its text is produced by replaying the same edits.

Two things go with it:

- an `integration.test.ts`, pinning the form under both orders of the configuration, by the precedent of `declaration-colon-space-after`;
- a paragraph in the README saying in what form the added solidus is written, or the example that turns `1.5` into `3 / 2` becomes false under a `never`.

Filed as a bug rather than as an enhancement because the plugin has the machinery for a writer to read its neighbours — it was built for the semicolon writer over the whole of #356 — and this rule writes a delimiter while reading nobody. Left as it is, the constant stops being merely unconfigurable the day #548 lands and becomes order-dependent: a `value-slash-space-before: never` listed ahead of this rule would be left with a warning for the run after, which is the `hard` class of #356 all over again.
